styles.css

```diff
@@ -27,6 +27,34 @@ body {
   display: none;
 }
 
+.show-hamburger {
+  margin: 0;
+  padding-top: 100px;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: #6070ffa8;
+}
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161509772-8d06045b-1c2e-429a-a19f-5a13841832a9.png) |  ![Screenshot_523](https://user-images.githubusercontent.com/69638013/201870382-7a504cd4-4697-4810-96ff-e31804bfe4a6.png)


- [ ] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 
- [ ] It would be better for the background not to be scrollable when the mobile nav is open 👍🏿

Reply:
Issues have been fixed as requested. Thank you for your review